cmake/SetupPackages.cmake

```diff
@@ -91,6 +91,11 @@ if (RAJA_ENABLE_TESTS)
       googletest
       GIT_REPOSITORY https://github.com/google/googletest.git
       GIT_TAG release-1.7.0
+      CMAKE_ARGS                
+          -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
```

Comment:
If a C or CXX compiler is not explicitly specified in the `CMAKE_XXX_COMPILER` variables, does this still work?

I was doing something similar yesterday and it just tried to pass an empty string. We might need to check the variable and build up the `CMAKE_ARGS` list depending on what is set.